nilearn/plotting/img_plotting.py

```diff
@@ -1996,25 +2089,29 @@ def plot_carpet(img, mask_img=None, detrend=True, output_file=None,
         (int(data.shape[0] + 1) // 10, int(data.shape[0] + 1) // 5, 1))
     xticks = list(range(0, data.shape[0])[::interval])
     axes.set_xticks(xticks)
+    axes.set_xlabel("time (s)")
 
-    axes.set_xlabel('time (s)')
-    axes.set_ylabel('voxels')
     if title:
         axes.set_title(title)
+
     labels = tr * (np.array(xticks))
     labels *= (2 ** n_decimations)
     axes.set_xticklabels(['%.02f' % t for t in labels.tolist()])
 
     # Remove and redefine spines
-    for side in ['top', 'right']:
+    for side in ["top", "right"]:
```

Comment:
(Nitpicking) @kchawla-pi had told me that single quote should be preferred to double quotes IIRC.

Reply:
Sorry, I normally use `black` to automate my formatting, and it uses double quotes. I can switch.